plugin-viewport/src/main/kotlin/com/mapbox/maps/plugin/viewport/ViewportExt.kt

```diff
@@ -0,0 +1,15 @@
+@file:JvmName("ViewportUtils")
+
+package com.mapbox.maps.plugin.viewport
+
+import com.mapbox.maps.MapboxExperimental
+import com.mapbox.maps.plugin.Plugin
+import com.mapbox.maps.plugin.delegates.MapPluginProviderDelegate
+
+/**
+ * Extension val for MapView to get the [ViewportPlugin] instance.
+ */
+@MapboxExperimental
+val MapPluginProviderDelegate.viewport: ViewportPlugin
```

Comment:
I remember it was named `experimentalViewport` at some point, just noting it here (although I do not remember why it was done like that but I like current solution more).

Reply:
experimentalViewport is another package that's used for prototyping from scratch in parallel with the initial version, and has been promoted in #1032

Reply:
This should be the final API that users see.